lib/gooddata/rest/connection.rb

```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
 
         # Reset old cookies first
         if options[:sst_token]
-          merge_headers!(:x_gdc_authsst => options[:sst_token])
+          headers = {
+            :x_gdc_authsst => options[:sst_token],
+            :x_gdc_authtt => options[:tt_token]
+          }
+          merge_headers!(headers)
           get('/gdc/account/token', @request_params)
 
           @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
           GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
           @auth = {}
           refresh_token :dont_reauth => true
+        elsif  options[:headers][:x_gdc_authsst]
+          @request_params = options[:headers]
+          @user = get('/gdc/app/account/bootstrap')['bootstrapResource']
+          GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
```

Comment:
Calls 'gooddata.logger.info("connected using sst to server #{@server.url} to profile \"#{@user['accountsetting']['login']}\"")' 2 times (DuplicateMethodCall)